File: Programs/Reverse_The_Array.py
""" CodingNinjas """
def reverseArray(arr, m):
    i= m+1
    j= len(arr) - 1

    while i<j:
        arr[i], arr[j]= arr[j], arr[i]

        i += 1
        j -= 1
        
    return arr

""" GeeksForGeeks """
class Solution:
    def reverseArray(self, arr):
        i, j= 0, len(arr)-1
        while i < j:
            arr[i], arr[j]= arr[j], arr[i]
            i += 1
            j -= 1

File: Programs/Reverse_The_Array.cpp
#include <bits/stdc++.h> 
using namespace std;

/* CodingNinjas */
void reverseArray(vector<int>& arr, int m) {
    for(int i= m+1, j= arr.size()-1; i<j; i++, j--)
        swap(arr[i], arr[j]);
}

/* GeeksForGeeks */
class Solution {
  public:
    void reverseArray(vector<int>& arr) {
        for(int i=0, j= arr.size()-1; i<j; i++, j--)
            swap(arr[i], arr[j]);
    }
};
